subprojects/docs/src/docs/userguide/userguide.adoc

```diff
@@ -14,55 +14,35 @@
-    <div class="technology-logo logo-javascript"><svg width="85px" height="85px" viewBox="0 0 85 85" version="1.1" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient x1="100%" y1="0%" x2="0%" y2="100%" id="linearGradient-7"><stop stop-color="#3ECFE0" offset="0%"></stop><stop stop-color="#4CA5AF" offset="100%"></stop></linearGradient></defs><g stroke="none" stroke-width="1" fill="none" fill-rule="nonzero"><g transform="translate(5.000000, 6.000000)"><rect fill="url(#linearGradient-7)" x="0" y="0" width="74" height="74"></rect><path d="M49.6936995,56.9169364 C51.1899405,59.3175189 53.1365866,61.0820396 56.5794737,61.0820396 C59.4717347,61.0820396 61.3193389,59.6616178 61.3193389,57.6989795 C61.3193389,55.3470575 59.4210347,54.5140369 56.2375431,53.1457512 L54.4925182,52.4100515 C49.4555271,50.3014318 46.1093238,47.6598643 46.1093238,42.0754978 C46.1093238,36.9313926 50.0981208,33.0153846 56.3318688,33.0153846 C60.7698918,33.0153846 63.9604578,34.5331274 66.2596461,38.5070645 L60.8241291,41.9364679 C59.6273721,39.8278482 58.3362894,38.9971448 56.3318688,38.9971448 C54.2873598,38.9971448 52.9915609,40.2715852 52.9915609,41.9364679 C52.9915609,43.99411 54.2885389,44.8271307 57.2833791,46.1015711 L59.028404,46.8361123 C64.9591309,49.3351742 68.3076923,51.8828966 68.3076923,57.6109272 C68.3076923,63.7861706 63.3709223,67.1692308 56.7410064,67.1692308 C50.2584744,67.1692308 46.0704145,64.1337453 44.0211892,60.1551738 L49.6936995,56.9169364 Z M25.035789,57.5112891 C26.132325,59.4229498 27.129819,61.0391721 29.5280493,61.0391721 C31.8213422,61.0391721 33.2680623,60.157491 33.2680623,56.7292461 L33.2680623,33.4069854 L40.2481622,33.4069854 L40.2481622,56.8219327 C40.2481622,63.9240419 36.0105813,67.1564864 29.8251751,67.1564864 C24.2363789,67.1564864 20.9998292,64.3144841 19.3538462,60.8914528 L25.035789,57.5112891 Z" fill="#FFFFFF"></path></g></g></svg></div>
-</div>
-++++
+image::userguide-languages.png[]
 
-== New projects with Gradle
+== Supported IDEs
+Gradle is supported on Android Studio, IntelliJ IDEA, Eclipse Buildship, and Netbeans.
```

Comment:
Hmmm I know the existing page uses the word "support" but I'm not sure it's the correct word to use here. Gradle has integrations with these editors to provide a better/more seem-less experience.

I also think its worth still worth mentioning the CLI. In fact not only mentioning it but saying that its robust/comprehensive/fully-featured or something like that. There are people who prefer using command-line and we want to let them know there is good support for their use-cases

Reply:
Updated in latest commit.